src/components/ConditionsEditor/ConditionsGroup.vue

```diff
@@ -140,24 +143,28 @@ export default class ConditionsGroup extends Vue {
     newGroups.push({
       operator: 'and',
       // Pass undefined values to force ConditionForm to use its default condition prop value
-      conditions: [undefined, undefined],
+      conditions: [undefined],
       groups: [],
     });
 
     const newConditionsTree = {
       ...this.conditionsTree,
       groups: newGroups,
-    };
-    this.emitUpdatedConditionTree(newConditionsTree);
+    } as AbstractFilterTree;
+    // When there is only one condition, the operator is an empty string.
+    // Set the operator to 'and' when a group is added.
```

Comment:
```suggestion
```

Reply:
We can remove this comment :) There are already in `setOperatorIfNecessaryAndUpdateConditionTree `